lib/ReportDataCollector.php

```diff
@@ -203,20 +204,41 @@ private function getSystemConfigDetailArray() {
 	}
 
 	/**
+	 * Sanitize values from the given hash array by removing
+	 * sensitive values
+	 *
+	 * @param array $values hash array
+	 * @return array sanitized array
+	 */
+	private function sanitizeValues($values) {
+		foreach($values as $key => $value) {
+			if (stripos($key, 'password') !== FALSE) {
+				$values[$key] = \OCP\IConfig::SENSITIVE_VALUE;
+			}
+		}
+		return $values;
+	}
+
+	/**
+	 * @return array
+	 */
+	private function getCoreConfigArray() {
+		// Get core config data
+		$appConfig = $this->appConfig->getValues('core', false);
+		// this one is reported separately already
+		unset($appConfig['oc.integritycheck.checker']);
+		return $this->sanitizeValues($appConfig);
+	}
+	/**
 	 * @return array
 	 */
 	private function getAppsDetailArray() {
 		// Get app data
 		foreach($this->apps as &$app) {
-			if($app['active']) {
+			if($app['active'] || $app['id'] === 'core') {
```

Comment:
it doesn't, but thanks for pointing at this. it was my previous attempt that didn't work.

will remove